### PR TITLE
[KOGITO-679] Provide commands to create/delete infinispan instance

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-679

Smoke tests ran ok after change in infrastructure.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: cmd/kogito/command/install/infinispan.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+		return err
+	}
+
+	if installed, err := shared.SilentlyInstallOperatorIfNotExists(i.flags.namespace, "", i.Client); err != nil {
+		return err
+	} else if !installed {
+		return nil
+	}
+
+	install := shared.ServicesInstallationBuilder(i.Client, i.flags.namespace).InstallInfinispan()
```
Comment (ricardozanini) [inline cmd/kogito/command/install/infinispan.go]: why not just:

```go
return shared.ServicesInstallationBuilder(i.Client, i.flags.namespace).InstallInfinispan().GetError()
```

File: cmd/kogito/command/remove/infinispan.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	i.command.Flags().StringVarP(&i.flags.namespace, "project", "p", "", "The project name where the operator will be deployed")
+}
+
+func (i *removeInfinispanCommand) Exec(cmd *cobra.Command, args []string) error {
+	var err error
+	if i.flags.namespace, err = shared.EnsureProject(i.Client, i.flags.namespace); err != nil {
+		return err
+	}
+
+	remove := shared.ServicesRemovalBuilder(i.Client, i.flags.namespace).RemoveInfinispan()
```
Comment (ricardozanini) [inline cmd/kogito/command/remove/infinispan.go]: Same here :)

File: cmd/kogito/command/remove/factory.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+package remove
+
+import (
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
+	"github.com/spf13/cobra"
+)
+
+// BuildCommands creates the commands available in this package
+func BuildCommands(ctx *context.CommandContext, rootCommand *cobra.Command) []context.KogitoCommand {
+	installCmd := newRemoveCommand(ctx, rootCommand)
```
Comment (sutaakar) [inline cmd/kogito/command/remove/factory.go]: Please rename the variable

Comment (radtriste) [reply cmd/kogito/command/remove/factory.go]: done :)

**Pull request merged**